app/src/test/kotlin/compiler/intermediate_form/UniqueIdentifierTest.kt

```diff
@@ -0,0 +1,51 @@
+package compiler.intermediate_form
+
+import kotlin.test.Test
+import kotlin.test.assertEquals
+class UniqueIdentifierTest {
```

Comment:
Maybe `UniqueIdentifierFactoryTest` would be a better name, as `UniqueIdentifier` is just a container for a string and obviously we don't test it here (because there is nothing to test).